src/containerapp/azext_containerapp/custom.py

```diff
@@ -2629,3 +2630,70 @@ def list_environment_telemetry_otlp(cmd,
 
     return containerapp_env_def
 
+
+def listreplica_containerappsjob(cmd, resource_group_name, name, execution=None):
+    if execution is None:
+        executions = ContainerAppsJobPreviewClient.get_executions(cmd=cmd, resource_group_name=resource_group_name, name=name)
+        execution = executions['value'][0]['name']
+        logger.warning('No execution specified. Using the latest execution: %s', execution)
+    try:
+        replicas = ContainerAppsJobPreviewClient.get_replicas(cmd, resource_group_name, name, execution)
+        return replicas['value']
+    except CLIError as e:
+        handle_raw_exception(e)
+
+
+def stream_job_logs(cmd, resource_group_name, name, container, execution=None, replica=None, follow=False, tail=None, output_format=None):
+    if tail:
+        if tail < 0 or tail > 300:
```

Comment:
```suggestion
        if tail < 1 or tail > 300:
```

Reply:
This is the same as other log streaming apis.

Reply:
Isn't the output for `tail = 0` always empty? Lets fix it at other locations too